viewing_party/main.py

```diff
@@ -0,0 +1,133 @@
+# test_wave_01
+def create_movie(movie_title, genre, rating):
+    movie_dict = {}
+    if not movie_title or not genre or not rating:
+        return None
+    else:
+        movie_dict["title"] = movie_title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data
```

Comment:
This passes the test, but it replaces the watchlist with a new list consisting of just one movie.  This is a little dangerous.

```suggestion
def add_to_watchlist(user_data, movie):
    user_data["watchlist"].append(movie)
    return user_data
```